src/content/docs/en/guides/content.mdx

```diff
@@ -43,13 +43,21 @@ Write blog posts in your existing Content Management System (CMS) such as Storyb
 
 ## Managing content pages
 
+### Page files
+
 Markdown and MDX files that live in your `src/pages` directory will automatically generate pages on your site using Astro's [file-based routing](/en/guides/routing/), built at a URL corresponding to the post's file path. 
 
+### Local content
+
 You can also choose to keep your Markdown and MDX files outside of the `src/pages` directory, and instead [import their content](/en/guides/markdown-content/#importing-markdown) into `.astro` pages.
 
-Astro's own [content collections](/en/guides/content-collections/) provide a powerful, type-safe way to work with your Markdown, MDX, or Markdoc content.
+### Content collections
+
+Astro's own [content collections](/en/guides/content-collections/) provide a powerful, type-safe way to work with Markdown, MDX, or Markdoc content organized in the `src/content/` directory. 
+
+### Remote content
 
-If you're writing your content in a CMS, you can fetch your posts and use [dynamic routing](/en/guides/routing/#dynamic-routes) to use one `.astro` file to generate a route for each post. In Astro's default static mode, these routes are generated at build time. If you opt-in to [SSR mode](/en/guides/server-side-rendering/), you respond to a request at runtime and fetch the content on demand.
+You can also [fetch markdown remotely](https://docs.astro.build/en/guides/markdown-content/#fetching-remote-markdown) from a source outside of your project. This will require your own Markdown parser, or a community integration (e.g. [`astro-remote`](https://github.com/natemoo-re/astro-remote)).
```

Comment:
Fix the check link error because of the absolute link.
```suggestion
You can also [fetch markdown remotely](/en/guides/markdown-content/#fetching-remote-markdown) from a source outside of your project. This will require your own Markdown parser, or a community integration (e.g. [`astro-remote`](https://github.com/natemoo-re/astro-remote)).
```